src/lib/picker/category.component.ts

```diff
@@ -137,6 +135,7 @@ export class CategoryComponent implements OnChanges, OnInit, AfterViewInit {
   @ViewChild('container', { static: true }) container!: ElementRef;
   @ViewChild('label', { static: true }) label!: ElementRef;
   containerStyles: any = {};
+  emojisToDisplay: any[]  = []
```

Comment:
emojis is working as an input + changed inside of the component, I decided not to change this flow right now, but introduce one more value to rely on in template